packages/cli/src/lib/snapshot/snapshot.ts

```diff
@@ -28,4 +47,53 @@ export class Snapshot {
   async delete() {
     // TODO: CDX-359: Delete snapshot once previewed
   }
+
+  get lastestReport(): ResourceSnapshotsReportModel {
+    if (this.model.reports === undefined) {
+      throw new Error(
+        `No detailed report found for the snapshot ${this.snapshotId}`
+      );
+    }
+    return this.model.reports.slice(-1)[0];
+  }
+
+  private get snapshotId() {
+    return this.model.id;
+  }
+
+  private async refreshSnapshotData() {
+    this.model = await (this.client.get as any)(this.model.id, {
```

Comment:
For the `as any` cast, is this a bug in platform-client ? Should this be a TODO ?

Reply:
yes, a  bug in the platform-client. I created a Pull request and added the link to the the jira in the comments

Reply:
Pull request : https://github.com/coveo/platform-client/pull/322